python_modules/dagster/dagster_tests/cli_tests/command_tests/assets.py

```diff
@@ -37,6 +37,21 @@ def single_run_partitioned_asset() -> None: ...
 def multi_run_partitioned_asset() -> None: ...
 
 
+class MyConfig(Config):
+    some_prop: str
+
+
+@asset
+def asset_with_config(context: AssetExecutionContext, config: MyConfig):
+    context.log.info(f"some_prop:{config.some_prop}")
+
+
+@asset
+def asset_assert_with_config(context: AssetExecutionContext, config: MyConfig):
+    assert config.some_prop == "foo"
+    context.log.info(f"some_prop:{config.some_prop}")
```

Comment:
FYI, adding the two assets here affected the test in:

```
python_modules/dagster/dagster_tests/cli_tests/command_tests/test_asset_list_command.py
```

So that test was updated as well. Let me know if you'd prefer an alternate approach.

<img width="445" alt="image" src="https://github.com/user-attachments/assets/0f5361bf-2128-4c57-9b7c-a944255ca5ab" />